beater/test_approved_stream_result/TestRequestIntegrationSuccess.approved.json

```diff
@@ -0,0 +1,3 @@
+{
+    "accepted": 4
+}
```

Comment:
We should keep one test ensuring the current result stays the same, by not having the query param set.

Reply:
yes perfect, I overlooked that!